### PR TITLE
Update archive to update flathub screenshots

### PR DESCRIPTION
**Pull request by mhogomchungu**

### CODE REVIEW

File: io.github.mhogomchungu.media-downloader.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
                 {
                      "type": "archive",
                       "url": "https://github.com/mhogomchungu/media-downloader/releases/download/4.9.0/media-downloader-4.9.0.1.tar.xz",
-                      "sha256": "f078da3cd5aecb7c221c4bb51e5af4174f384f6d85ea498100a7d729fbdddc58"
+                      "sha256": "c1e33ebe56500c1699b8b49382a816a1df66b18dbd12eb91f80a44ad0b88d9c2"
```
Comment (hfiguiere) [inline io.github.mhogomchungu.media-downloader.json:24]: Retaggibng is bad. This should have been a new tarball with a new version.

Comment (mhogomchungu) [reply io.github.mhogomchungu.media-downloader.json:24]: Yes, it is bad. I am totally new to flathub and the retag was to fix something specific to application's page on flathub.

It wont happen again.

Comment (hfiguiere) [reply io.github.mhogomchungu.media-downloader.json:24]: it's not flathub specific either.

Comment (mhogomchungu) [reply io.github.mhogomchungu.media-downloader.json:24]: Latest released version is here: https://github.com/mhogomchungu/media-downloader/releases/tag/4.9.0

Archive "media-downloader-4.9.0.tar.xz" is the official release version for 4.9.0.

Archive "media-downloader-4.9.0.1.tar.xz" is a temporary archive i am using 
 specifically to build on flathub while understanding how flatbub works. 

Screenshots on flathub seems to come from the archive and the latest change of the archive was to change screenshots taken on windows 10 as seen [here](https://flathub.org/apps/io.github.mhogomchungu.media-downloader) with screenshots taken on plasma 5. 

Once i have everything on flathub sorted out and i think i am here already, i will make a new release version 5.0.0 and this version will be the first one to officially support flatpack and flathub.

**Pull request merged**